.github/workflows/coverage.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches: [master]
+  pull_request:
+
+name: Code Coverage
+
+jobs:
+  coverage:
+    name: Test Coverage
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: checkout source
+        uses: actions/checkout@v2
+
+      - name: Install nightly toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          profile: minimal
+          override: true
+
+      - name: Run cargo-tarpaulin
+        uses: actions-rs/tarpaulin@v0.1
+        with:
+          version: '0.13.2'
```

Comment:
As discussed, I'm not 100% comfortable bringing coverage metric right now since my  end to end tests rely on `assert_cmd` for which coverage isn't computed, causing the coverage metric to be way off for `src/main.rs`. Maybe in a second phase, once I've replaced the `assert_cmd` use with calls to Clap's main.

Reply:
fair enough